AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -84,42 +82,80 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
     private checkForExistingRequest() {
         this.logger.debug(`${this.loggerPrefix} Checking for existing hearing`);
         this.hearing = this.hearingService.getCurrentRequest();
+        this.otherInformationDetails = OtherInformationModel.init(this.hearing.other_information);
     }
 
     private initForm() {
-        const find_judge = this.hearing.participants.find(x => x.is_judge === true);
-        if (!find_judge) {
-            this.judge = new JudgeResponse({ email: this.constants.PleaseSelect, display_name: '' });
-        } else {
-            this.logger.debug(`${this.loggerPrefix} Found judge in hearing. Populating existing selection.`);
-            this.judge = AssignJudgeComponent.mapJudge(find_judge);
-            this.canNavigate = true;
-        }
-        this.judgeDisplayName = new FormControl(this.judge.display_name, {
-            validators: [Validators.required, Validators.pattern(Constants.TextInputPattern), Validators.maxLength(255)],
-            updateOn: 'blur'
-        });
+        const existingJudge = this.hearing.participants.find(x => x.is_judge);
+
+        this.populateFormFields(existingJudge);
 
         this.form = this.fb.group({
-            judgeName: [this.judge.email, Validators.required],
-            judgeDisplayName: this.judgeDisplayName
+            judgeName: [this.judge.display_name, Validators.required],
+            judgeDisplayNameFld: this.judgeDisplayNameFld,
+            judgeEmailFld: this.judgeEmailFld,
+            judgePhoneFld: this.judgePhoneFld
         });
 
+        this.updateJudgeSummary();
+    }
+
+    updateJudgeSummary() {
         this.$subscriptions.push(
             this.judgeName.valueChanges.subscribe(judgeUserId => {
                 this.addJudge(judgeUserId);
                 this.isJudgeSelected = judgeUserId !== null;
+                this.otherInformationDetails.judgeEmail = '';
+                this.judgeEmailFld.setValue('');
+                this.otherInformationDetails.judgePhone = '';
+                this.judgePhoneFld.setValue('');
                 this.canNavigate = this.isJudgeSelected;
             })
         );
 
         this.$subscriptions.push(
-            this.judgeDisplayName.valueChanges.subscribe(name => {
+            this.judgeDisplayNameFld.valueChanges.subscribe(name => {
```

Comment:
I'd consider just subscribing to this.form.valueChanges here instead of all the separate ones